docs/en-US/component/message.md

```diff
@@ -106,29 +106,30 @@ const { appContext } = getCurrentInstance()!
 ElMessage({}, appContext)
 ```
 
-## Message API
+## API
 
 ### Options
```

Comment:
## Suggest

`Attributes`

https://element-plus.org/en-US/component/link.html#attributes

Reply:
Since this is a command or function component, options are reserved.